src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,6 +17,18 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+# Set to "yes" in order to ignore diff errors and save results to update test
+getjson="no"
```

Comment:
Only getjson is used for debugging purposes.  The termwidth comes into play if a test fails to see the comparison of the json.  I wouldn't change to diff_opts since there are diffs that don't use -y.

I'll put in additional description for getjson.